source/material_model/rheology/visco_plastic.cc

```diff
@@ -531,6 +567,14 @@ namespace aspect
                            "Select what type of viscosity law to use between diffusion, "
                            "dislocation, frank kamenetskii, and composite options. Soon there will be an option "
                            "to select a specific flow law for each assigned composition ");
+        prm.declare_entry ("Use minimum of diffusion and dislocation creep viscosity", "false",
+                           Patterns::Bool(),
+                           "Whether to take the minimum of the diffusion and dislocation creep "
+                           "viscosity (true) or use a composite of both viscosities (false). "
+                           "When the full strain rate is used to compute each viscosity "
```

Comment:
Since we don't properly distribute the strain rate between dislocation and diffusion creep (like it's done in diffusion_dislocation.cc), but use the full strain rate to compute both the dislocation and the diffusion viscosity, it's better to take the minimum of the two.